akka-testkit/src/test/resources/reference.conf

```diff
@@ -31,4 +31,15 @@ akka {
       }
     }
   }
+
+  circuit-breaker {
+    identified {
+      max-failures = 1
+      call-timeout = 100 ms
+      reset-timeout = 200 ms
+      exception-allowlist = [
+        "akka.pattern.CircuitBreakerSpec$AllowException"
+      ]
+    }
+  }
```

Comment:
I would prefer to include this in `CircuitBreakerSpec` instead. It's too hidden here.

```
class CircuitBreakerSpec extends AkkaSpec("""
  akka.circuit-breaker {
     identified {
       max-failures = 1
       call-timeout = 100 ms
       reset-timeout = 200 ms
       exception-allowlist = [
         "akka.pattern.CircuitBreakerSpec$AllowException"
       ]
     }
   }
   """) {
```

Reply:
Done ✅